elements/pfe-navigation/demo/index.html

```diff
@@ -18,9 +18,9 @@
   <link href="../pfe-navigation--lightdom.css" rel="stylesheet">
 
   <!-- uncomment the es5-adapter if you're using the umd version -->
-  <script src="/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
-  <script src="/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-  <script src="/requirejs/require.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/2.2.10/custom-elements-es5-adapter.js"></script>
```

Comment:
Why would we use the CDN for the polyfills instead of our locally version-controlled copies?

Reply:
I would prefer this be reverted to using the local copies.  It ensures we are all using the same version as we test our components and communicates to consumers what versions work with our components.

Reply:
@castastrophe - We need this for Netlify. Until we figure out how to get Netlify to use the local web components files, this is the only work around that I can come up with. If I don't have those changes, then this doesn't work.

https://deploy-preview-421--happy-galileo-ea79c4.netlify.com/elements/pfe-navigation/demo/